job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/Constants.groovy

```diff
@@ -44,4 +44,7 @@ class Constants {
     static final String CEKIT_CACHE_LOCAL = ""
     static final String VERBOSE	= "false"
     static final String LOCAL_MVN_REP = "/home/jenkins/.m2/repository"
+    static final String GITHUB_REPO_PREFIX = "https://www.github.com/kiegroup/"
+    static final String REPO_FILE_PATH= "droolsjbpm-build-bootstrap/script/repository-list.txt"
```

Comment:
are they used anywhere?

Reply:
This is weird. I don't have it in the source code. I removed it... Hm...

Reply:
we already have defined env.variable GITHUB_KIEGROUP_PREFIX, so why we have agaiin GITHUB_REPO_PREFIX here? we probably can hide this file in REPO_FILE_PATH, but that is just side comment